tests/python/contrib/test_random.py

```diff
@@ -149,5 +149,6 @@ def test_rpc(dtype):
 if __name__ == "__main__":
     test_randint()
     test_uniform()
-    test_normal()
+    # TODO(trevmorr): Disabled in neo-ai/tvm due to MemoryError
```

Comment:
You need to annotate the test with "pytest.skip" as well to skip the test. Otherwise, it will be still picked up by pytest.

Reply:
Got it, thanks!